javascript/sdk/lib/types.ts

```diff
@@ -271,6 +277,7 @@ export interface ActivityQuery {
   statuses?: string;
   tags?: string;
   control?: ControlCode;
+  impersonate?: string;
```

Comment:
i'd expect some deletion code somewhere since impersonate is no longer in the headers? not a bug/ just possible dead code somewhere